play/binary-explorer/index.html

```diff
@@ -121,6 +125,32 @@ <h4 class="modal-title">Decimal to Letter Conversions</h4>
 			</div>
 		</div>
 
+		<!-- Options Modal -->
+		<div class="modal fade" id="options" role="dialog">
+			<div class="modal-dialog">
+				<div class="modal-content">
+					<div class="modal-header">
+						<h4 class="modal-title">Options</h4>
+						<button type="button" class="close" data-dismiss="modal">&times;</button>
+					</div>
+
+					<div class="modal-body">
+						<p>Select the way you would like to copy your message.</p>
+						<form>
+							<input type="radio" id="character" name="copy" value="character" checked>
+							<label for="character">Copy Message using Characters</label><br>
+							<input type="radio" id="binary" name="copy" value="binary">
```

Comment:
Could you change this to say something like "Copy the Binary Message" or something like that. I think that this is not immediately clear to a person. Maybe even show an example so "Copy the Binary Message (ex: 101010101)".